examples/analytics-reportWhen.amp.html

```diff
@@ -0,0 +1,82 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <title>AMP Analytics reportWhen</title>
+  <link rel="canonical" href="analytics.amp.html" >
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-custom>
+    .box {
+      background: #ccc;
+      border: 1px solid #aaa;
+      padding: 10px;
+      margin: 10px;
+      display: inline-block;
+      min-width: 300px;
+      text-align: center;
+    }
+  </style>
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <script async custom-element="amp-anim" src="https://cdn.ampproject.org/v0/amp-anim-0.1.js"></script>
+  <script async custom-element="amp-analytics" src="https://cdn.ampproject.org/v0/amp-analytics-0.1.js"></script>
+  <script async custom-element="amp-video" src="https://cdn.ampproject.org/v0/amp-video-0.1.js"></script>
+  <script>
+    function mockUnload() {
+      document.dispatchEvent(new Event('unload'));
+    }
+  </script>
+</head>
+<body>
+<h1 id="top">AMP Analytics reportWhen</h1>
+<div>
+  An event will be triggered once the image below becomes visible. But it won't be reported until the document exits
+  (e.g. an unload or pagehide event is fired). Look in the developer console, and then scroll to the bottom of the page
+  to show the image, and finally scroll back up and click the button just below this text. The network request will not
+  appear until the button is clicked.
```

Comment:
Aside from the above paragraph, and of course the addition of reportWhen in the amp-analytics config, this file is largely the same as the existing analytics.amp.html, with large sections removed.